introduction.md

```diff
@@ -1,3 +1,8 @@
 Hello, my name is james
 
-My favorite codeschool path is JavaScript
\ No newline at end of file
+My favorite codeschool path is JavaScript
+
+Deadly Skills
+==========
+* Face biting
+* Uno playing
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?